pages/azavea-test/index.jsx

```diff
@@ -0,0 +1,14 @@
+// actions
+// None at the moment
+
+// Layout
+// We could put all our markup straight into this Page, but given that we may end up with multiple
+// sub-pages inside of this page, it seems like the recommended thing to do is to wrap in a Layout.
+// But currently this seems kind of boiler-plateish to me, so if it stays that way after we have
+// some more complex content then we should probably refactor so that this file actually does
+// something useful.
```

Comment:
This makes sense. From my read of this, it seems like this is fairly analogous to our concept of `screens` in DistrictBuilder. I can see how it might be useful to have different layouts on a single page, it also could be useful to handle the import of components used in the page primarily here so that they could be shared across different layouts.

Reply:
Yeah, looking around a little it seems like there are a lot of `pages` that are just tiny wrappers around `Layout` components like this, but then some that have more going on.  Preserving the basic `pages` structure but defining single-use components in the same file rather than always putting them in `layout` files would seem like a sensible thing to consider.  Seems like the page component will be the starting point regardless for changing a page, so it won't be confusing to have some of the pieces right there rather than broken out.  And if something starts there but then needs to be shared, it would be easy to break it out when that becomes clear.